x-pack/plugin/security/src/internalClusterTest/java/org/elasticsearch/integration/ClusterPrivilegeIntegrationTests.java

```diff
@@ -79,8 +79,9 @@ protected String configRoles() {
 
     @Override
     protected String configUsers() {
-        final String usersPasswdHashed = new String(Hasher.resolve(
-            randomFrom("pbkdf2", "pbkdf2_1000", "bcrypt", "bcrypt9")).hash(new SecureString("passwd".toCharArray())));
+        final Hasher passwdHasher = inFipsJvm() ? Hasher.resolve(randomFrom("pbkdf2", "pbkdf2_1000")) :
+            Hasher.resolve(randomFrom("pbkdf2", "pbkdf2_1000", "bcrypt", "bcrypt9")) ;
+        final String usersPasswdHashed = new String(passwdHasher.hash(SecuritySettingsSourceField.TEST_PASSWORD_SECURE_STRING));
```

Comment:
The password was `passwd` which is different from `SecuritySettingsSourceField.TEST_PASSWORD_SECURE_STRING`. This and a few similar changes in other places are causing CI failure for this PR. I think the fix should be updating `AbstractPrivilegeTestCase#setUser` to use the new password as well.